### PR TITLE
Images

### PR DESCRIPTION
**Pull request by srishtigautam21**
Done with image component and the home page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +34,17 @@
             </li>
         </ul>
     </nav>
+    <div class="landing-page-container">
+        <h1 class="lg-header">Built your web projects faster and better</h1>
+        <h3 class="mdm-header">Umbrella is a lightweight and modular CSS library,made with predefined styled
```
Comment (humzahasan) [inline index.html:39]: According to standards for medium we use md, so consider renaming to md instead of mdm

**Review by humzahasan — APPROVED** [review]
- Code looks good.

- Consider code for refactoring.

Approved.

File: index.html (hunk 1 above)
Comment (srishtigautam21) [reply index.html:39]: ok didn't knew that will make the changes

**Review by dcs-soni — APPROVED** [review]
Looks good to merge, all the class names and structure looks fine to me 👍 Just a suggestion, you can use prettier to format the code.

**Comment by archit7448** [comment]
look good to merge

**Review by archit7448 — APPROVED** [review]

**Comment by itsmebipul00** [comment]
Great work! Good to merge

**Pull request merged**